clients/java/src/main/java/io/lakefs/clients/api/model/User.java

```diff
@@ -111,6 +115,29 @@ public void setFriendlyName(String friendlyName) {
   }
 
 
+  public User email(String email) {
+    
+    this.email = email;
+    return this;
+  }
+
+   /**
+   * Get email
+   * @return email
+  **/
+  @javax.annotation.Nullable
+  @ApiModelProperty(value = "")
+
+  public String getEmail() {
+    return email;
+  }
+
+
+  public void setEmail(String email) {
+    this.email = email;
```

Comment:
no verification? or do we do it in a different place?

Reply:
Currently no verification, we should add the verification once we [support sending emails](https://github.com/treeverse/lakeFS/issues/3103).